---a/net/minecraft/tileentity/TileEntityEnchantmentTable.java
+++b/net/minecraft/tileentity/TileEntityEnchantmentTable.java
@@ -1,6 +1,7 @@
 package net.minecraft.tileentity;
 
 import java.util.Random;
+
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.InventoryPlayer;
 import net.minecraft.inventory.Container;
@@ -13,8 +13,8 @@
 import net.minecraft.util.MathHelper;
 import net.minecraft.world.IInteractionObject;
 
-public class TileEntityEnchantmentTable extends TileEntity implements IUpdatePlayerListBox, IInteractionObject
-{
+public class TileEntityEnchantmentTable extends TileEntity implements IUpdatePlayerListBox, IInteractionObject {
+
     public int tickCount;
     public float pageFlip;
     public float pageFlipPrev;
@@ -29,85 +29,68 @@
     private String customName;
     private static final String __OBFID = "CL_00000354";
 
-    public void writeToNBT(NBTTagCompound compound)
-    {
+    public void writeToNBT(NBTTagCompound compound) {
         super.writeToNBT(compound);
 
-        if (this.hasCustomName())
-        {
+        if (this.hasCustomName()) {
             compound.setString("CustomName", this.customName);
         }
     }
 
-    public void readFromNBT(NBTTagCompound compound)
-    {
+    public void readFromNBT(NBTTagCompound compound) {
         super.readFromNBT(compound);
 
-        if (compound.hasKey("CustomName", 8))
-        {
+        if (compound.hasKey("CustomName", 8)) {
             this.customName = compound.getString("CustomName");
         }
     }
 
-    public void update()
-    {
+    public void update() {
         this.bookSpreadPrev = this.bookSpread;
         this.bookRotationPrev = this.bookRotation;
-        EntityPlayer entityplayer = this.worldObj.getClosestPlayer((double)((float)this.pos.getX() + 0.5F), (double)((float)this.pos.getY() + 0.5F), (double)((float)this.pos.getZ() + 0.5F), 3.0D);
+        EntityPlayer entityplayer = this.worldObj.getClosestPlayer((double) ((float) this.pos.getX() + 0.5F), (double) ((float) this.pos.getY() + 0.5F), (double) ((float) this.pos.getZ() + 0.5F), 3.0D);
 
-        if (entityplayer != null)
-        {
-            double d0 = entityplayer.posX - (double)((float)this.pos.getX() + 0.5F);
-            double d1 = entityplayer.posZ - (double)((float)this.pos.getZ() + 0.5F);
-            this.field_145924_q = (float)Math.atan2(d1, d0);
+        if (entityplayer != null) {
+            double d0 = entityplayer.posX - (double) ((float) this.pos.getX() + 0.5F);
+            double d1 = entityplayer.posZ - (double) ((float) this.pos.getZ() + 0.5F);
+            this.field_145924_q = (float) Math.atan2(d1, d0);
             this.bookSpread += 0.1F;
 
-            if (this.bookSpread < 0.5F || field_145923_r.nextInt(40) == 0)
-            {
+            if (this.bookSpread < 0.5F || field_145923_r.nextInt(40) == 0) {
                 float f1 = this.field_145932_k;
 
-                do
-                {
-                    this.field_145932_k += (float)(field_145923_r.nextInt(4) - field_145923_r.nextInt(4));
-                }
-                while (f1 == this.field_145932_k);
+                do {
+                    this.field_145932_k += (float) (field_145923_r.nextInt(4) - field_145923_r.nextInt(4));
+                } while (f1 == this.field_145932_k);
             }
-        }
-        else
-        {
+        } else {
             this.field_145924_q += 0.02F;
             this.bookSpread -= 0.1F;
         }
 
-        while (this.bookRotation >= (float)Math.PI)
-        {
-            this.bookRotation -= ((float)Math.PI * 2F);
+        while (this.bookRotation >= (float) Math.PI) {
+            this.bookRotation -= ((float) Math.PI * 2F);
         }
 
-        while (this.bookRotation < -(float)Math.PI)
-        {
-            this.bookRotation += ((float)Math.PI * 2F);
+        while (this.bookRotation < -(float) Math.PI) {
+            this.bookRotation += ((float) Math.PI * 2F);
         }
 
-        while (this.field_145924_q >= (float)Math.PI)
-        {
-            this.field_145924_q -= ((float)Math.PI * 2F);
+        while (this.field_145924_q >= (float) Math.PI) {
+            this.field_145924_q -= ((float) Math.PI * 2F);
         }
 
-        while (this.field_145924_q < -(float)Math.PI)
-        {
-            this.field_145924_q += ((float)Math.PI * 2F);
+        while (this.field_145924_q < -(float) Math.PI) {
+            this.field_145924_q += ((float) Math.PI * 2F);
         }
 
         float f2;
 
-        for (f2 = this.field_145924_q - this.bookRotation; f2 >= (float)Math.PI; f2 -= ((float)Math.PI * 2F))
-        {
+        for (f2 = this.field_145924_q - this.bookRotation; f2 >= (float) Math.PI; f2 -= ((float) Math.PI * 2F)) {
             ;
         }
 
-        while (f2 < -(float)Math.PI)
-        {
-            f2 += ((float)Math.PI * 2F);
+        while (f2 < -(float) Math.PI) {
+            f2 += ((float) Math.PI * 2F);
         }
 
         this.bookRotation += f2 * 0.4F;
@@ -122,34 +122,27 @@
         this.pageFlip += this.field_145929_l;
     }
 
-    public String getName()
-    {
+    public String getName() {
         return this.hasCustomName() ? this.customName : "container.enchant";
     }
 
-    public boolean hasCustomName()
-    {
+    public boolean hasCustomName() {
         return this.customName != null && this.customName.length() > 0;
     }
 
-    public void setCustomName(String customNameIn)
-    {
+    public void setCustomName(String customNameIn) {
         this.customName = customNameIn;
     }
 
-    public IChatComponent getDisplayName()
-    {
-        return (IChatComponent)(this.hasCustomName() ? new ChatComponentText(this.getName()) : new ChatComponentTranslation(this.getName(), new Object[0]));
+    public IChatComponent getDisplayName() {
+        return (IChatComponent) (this.hasCustomName() ? new ChatComponentText(this.getName()) : new ChatComponentTranslation(this.getName(), new Object[0]));
     }
 
-    public Container createContainer(InventoryPlayer playerInventory, EntityPlayer playerIn)
-    {
+    public Container createContainer(InventoryPlayer playerInventory, EntityPlayer playerIn) {
         return new ContainerEnchantment(playerInventory, this.worldObj, this.pos);
     }
 
-    public String getGuiID()
-    {
+    public String getGuiID() {
         return "minecraft:enchanting_table";
     }
-}
+}
